diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7303f4cff..e72de51bf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,8 +20,11 @@ endif ()
 
 set (SWIG_ROOT ${PROJECT_SOURCE_DIR})
 
-set (SWIG_LIB share/swig/${SWIG_VERSION})
-
+if (WIN32)
+  set (SWIG_LIB bin/Lib)
+else ()
+  set (SWIG_LIB share/swig/${SWIG_VERSION})
+endif ()
 # Project wide configuration variables
 # ------------------------------------
 
@@ -80,7 +83,7 @@ if (WITH_PCRE)
   include_directories (${PCRE2_INCLUDE_DIRS})
 endif()
 
-if (WIN32)
+if (0)
   file (TO_NATIVE_PATH ${CMAKE_INSTALL_PREFIX}/${SWIG_LIB} SWIG_LIB_WIN_UNIX)
   string (REGEX REPLACE "\\\\" "\\\\\\\\" SWIG_LIB_WIN_UNIX "${SWIG_LIB_WIN_UNIX}")
 endif ()
